ipatests/test_integration/test_client.py

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (C) 2019  FreeIPA Contributors see COPYING for license
+#
+import random
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+
+class TestSSHFromClient(IntegrationTest):
+    """
+    Tests ssh from client when ldap_deref_threshold=0 is added to sssd.conf
+    Automated test for https://pagure.io/SSSD/sssd/issue/3979
+    """
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        tasks.install_master(cls.master)
+        cls.client = cls.clients[0]
+        tasks.install_client(cls.master, cls.client)
+
+    def test_ssh_from_client(self):
+        """
+        Test ssh from client after adding ldap_deref_threshold=0 to sssd.conf
+
+        Steps:
+        1. setup a master and a client
```

Comment:
No need for a client.

Reply:
changed it, to just include a master setup